uma/uma.go

```diff
@@ -564,6 +578,35 @@ func GetPayReqResponse(
 	}, nil
 }
 
+func getSignedCompliancePayeeData(
+	receivingVaspPrivateKeyBytes []byte,
+	payerIdentifier string,
+	payeeIdentifier string,
+	receiverChannelUtxos []string,
+	receiverNodePubKey *string,
+	utxoCallback string,
+) (*CompliancePayeeData, error) {
+	timestamp := time.Now().Unix()
+	nonce, err := GenerateNonce()
+	if err != nil {
+		return nil, err
+	}
+	payloadString := strings.Join([]string{payerIdentifier, payeeIdentifier, *nonce, strconv.FormatInt(timestamp, 10)}, "|")
```

Comment:
Should this also just use signablePayload() from the payreqresponse?

Reply:
ah its not super straightforward to reuse since we havent constructed payreqresponse at this point

Reply:
ooh yeah good point 🤔. Check out what we do in the kotlin sdk for `LnurlComplianceResponse`. It seems its signablePayload is there rather than at the root request level. Not sure whether that's possible here or not, but maybe?

Reply:
updated! will open a separate change to make the rest of the signature checks consistent